src/nanoemoji/color_glyph.py

```diff
@@ -421,7 +421,10 @@ def create(
         logging.debug(
             " ColorGlyph for %s (%s)", svg_filename or bitmap_filename, codepoints
         )
-        base_glyph = ufo.newGlyph(ufo_glyph_name)
+        if ufo_glyph_name not in ufo:
+            base_glyph = ufo.newGlyph(ufo_glyph_name)
+        else:
+            base_glyph = ufo[ufo_glyph_name]
```

Comment:
I don't understand this change. Under which circumstances can a UFO already contain glyph with given name, when we make sure that all color glyphs' names are unique?

Reply:
(encountered w/Bungee)